.github/workflows/ci.yaml

```diff
@@ -1,53 +1,24 @@
 name: CI
+
 on: [push]
 
 jobs:
-  lint:
-    name: Lint
-    runs-on: ubuntu-22.04
-    steps:
-
-    - name: Set up Go
-      uses: actions/setup-go@v4
-      with:
-        go-version: "1.21.1"
-
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v4
-
-    - name: lint
-      uses: golangci/golangci-lint-action@v3
-      with:
-        version: "v1.54.0"
-        working-directory: ${{matrix.module}}
-
   test:
-    name: Test
-
     runs-on: ubuntu-22.04
-
-    strategy:
-      matrix:
-        go:
-          - "1.21"
```

Comment:
I usually guarantee compatibility with older Go versions on my libs if possible, but I don't think this is a problem here =P

Reply:
Oh that's why you had the matrix setup... yeah, in our case testing with older Go versions is probably excessive.

Reply:
yeah it was...it wasn't just because Im a crazy motherfucker :rofl: . Some of my libs work even with Go 1.16...so I keep testing =P...but here it is unnecessary indeed

Reply:
So I would just add 1.22 on the matrix when it was released, and so on (coincidentally now it requires at least 1.21).